projects/wallet-template/README.md

```diff
@@ -1,2 +1,105 @@
-## Substrate Connect Wallet Template
+# Substrate Connect - Light Client Wallet Template
 
+## Overview
+
+Welcome to Substrate Connect - the ultimate light client wallet template! Substrate
+Connect leverages the power of the smoldot,
+a light client that implements the JSON RPC Spec to provide a highly resilient, 
+efficient, and user-friendly wallet experience. This template is designed to help 
+developers quickly build and deploy a decentralized wallet that interacts with 
+any blockchain in the Substrate ecosystem using a light client.
+
+What's more, when you use Substrate Connect as a browser extension, your dapps 
+can instantly connect to popular Substrate-based chains without the hassle of 
+waiting for synchronization. This is possible because Substrate Connect keeps 
+track of the chains in the background, even when the user isn't actively 
+connected to any dapps. At the same time, the dApp will show up as "connected" 
+within the extension, indicating that it is leveraging the extension's bootnodes 
+and specifications.
+
+## Getting Started
+
+### Prerequisites
+
+- **Corepack** - [Zero runtime dependency manager](https://github.com/nodejs/corepack)
```

Comment:
corepack just installs the correct version of the package manager. so if you have pnpm 9 and the project is using pnpm 8 you don't need to deliberately downgrade pnpm on your system. it can also work with other package managers like yarn or even just npm.